### DISCUSSION TITLE
How poac manage dependencies

### DISCUSSION THREAD

**Discussion by Mihawka** [cabinpkg/cabin#931] [Q&A] [answered]
Hi,
I'm curious about how poac detect and manage dependencies.

If I understand correctly I put in my poac.toml :
```
[dependencies]
"ToruNiina/toml11" = {git = "https://github.com/ToruNiina/toml11.git", tag = "v3.8.0"}

```
poac will pull the repo and what's he going to do next? 

Can you make an example with GLFW of how in theory poac will manage this dependency?

**Comment by ken-matsui** [comment, marked as answer]
Since GLFW has an include/ directory, Poac automatically passes the path to this directory when you build your project. However, Poac currently supports building a project with header-only dependencies. If you need to build GLFW to use, then it cannot be used. If it is a header-only library, then you can use it like `#inlcude <GLFW/glfw3.h>`, according to its directory structure.

So, as long as your dependency is header-only and provides its headers within the repository root (toml11 case) or include/ (GLFW case) directory, you can use it via Poac.
